dotnet/packages/Microsoft.TeamsAI/Microsoft.TeamsAI.Tests/StateTests/TurnStateTests.cs

```diff
@@ -0,0 +1,109 @@
+﻿using Microsoft.Teams.AI.State;
+using Microsoft.Teams.AI.Tests.TestUtils;
+using Microsoft.Bot.Schema;
+using Moq;
+using Microsoft.Bot.Builder;
+using Record = Microsoft.Teams.AI.State.Record;
+
+namespace Microsoft.Teams.AI.Tests.StateTests
+{
+    /// <summary>
+    /// TODO: Add tests that enforce required properties. Ex. If a required param/object is null, should throw exception.
+    /// </summary>
+    public class TurnStateTests
+    {
+        [Fact]
+        public async void Test_LoadState_NoStorageProvided_ShouldThrow()
+        {
+            // Arrange
+            var state = new TurnState<Record, Record, TempState>();
```

Comment:
The convention is to name variables referring to the turn sate as `turnState`. It doesn't matter for tests but for application code we should update (if you had implemented).